Examples/TodoApp/TodoApp/src/tasks/TasksInjector.swift

```diff
@@ -1,17 +1,25 @@
 import MobiusCore
+import MobiusExtras
 
-final class TasksInjector {
+final class MobiusControllerFactory {
```

Comment:
Yep, good point, I made it a struct but I'm keeping the fields private since they shouldn't be accessed outside, right?

Reply:
Sure, you can keep them private. The advantage of making them `internal` is that you can have the initializer generated for you, except that wouldn't work in this case since one of the fields is assigned statically